h2o-algos/src/main/java/hex/glm/GLM.java

```diff
@@ -464,6 +464,11 @@ public void init(boolean expensive) {
         case gaussian:
 //          if (_nclass != 1) error("_family", H2O.technote(2, "Gaussian requires the response to be numeric."));
           break;
+        case fractionalbinomial:
+          Vec resp = (_parms.train()).vec(_parms._response_column);
```

Comment:
you need to use `train()` instead of `_parms.train()` because the original train frame and the adapted ones might be different

Reply:
Good point.  Let me change it.